### ISSUE TITLE
Add Dockerize version of the API

### ISSUE THREAD

**Issue by angelocordon** [codebuddies/frontend#92]
When working on the frontend and for testing, it would be helpful to have a deployable, stable version of the API locally. This is so that we can ensure that our API endpoints are working as expected from the frontend's perspective as well as when testing, we can have a working API endpoint for e2e tests.

**Comment by billglover** [comment]
It is currently possible to run any version of the API locally for testing. I would like to focus discussion on what the expectations are for a stable API and how change should be approached.

**Comment by lpatmo** [comment]
I think we can close out this issue! I recorded a quick video here on how to get the local version of the API up and running with the frontend simultaneously: https://www.youtube.com/watch?v=t43vYwxzG-E

Re: expectations for a stable API -- we covered our expectations for what we need for the MVP in a late March meeting (https://github.com/codebuddies/backend/issues/94#issuecomment-602261487), and also agreed to meet to plan all the tickets for a feature before we work on the next feature. 

In summary, the current sprint involves:
- user login/registration (DONE) + password resetting and email confirmation validation (NOT DONE/HELP WANTED)
- create, edit, delete, and view resources against the API (DONE)
- only logged-in users can edit, create, and delete resources (DONE)
- implement create, edit, and delete on front-end (IN PROGRESS)
- implement client-side form validation on the front-end (IN PROGRESS)
